.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like we need to add the `.vscode` folder to the `.gitignore` file. This file doesn't need to be committed to github